Autopilot/AttitudeManager/Network/Datalink.h

```diff
@@ -34,12 +29,30 @@
 
 #define UHF_KILL_TIMEOUT 10000
 
-typedef enum _p_priority {
-    PRIORITY0 = 0,
-    PRIORITY1 = 1,
-    PRIORITY2 = 2,
-} p_priority;
+/**
+ * Different packet types that we can send over via the downlink
+ */
+typedef enum {
+    PACKET_TYPE_POSITION = 0,
+    PACKET_TYPE_STATUS = 1,
+    PACKET_TYPE_GAINS = 2,
+    PACKET_TYPE_CHANNELS = 3
+} PacketType;
 
+/**
+ * Defines the normal order that packets will be send down.  Note that this list contains packet types that should continually
+ * be sent down. It should NOT contain packets types that should be sent down from
+ * an event, ie gains. For these types of packets, the queuePacketType() method should
+ * be called to move the packet type to the front of the queue
+ */
+static const uint8_t DEFAULT_PACKET_ORDER[] = {
+    PACKET_TYPE_POSITION,
+    PACKET_TYPE_POSITION,
+    PACKET_TYPE_POSITION,
+    PACKET_TYPE_STATUS,
+    PACKET_TYPE_POSITION,
+    PACKET_TYPE_CHANNELS
+};
```

Comment:
So that we get status and channel updates about once every second. I put the position in the middle of those two so that we also get the most updated positional information most of the time,